### ISSUE TITLE
Training and Validation accuracy

### ISSUE THREAD

**Issue by ngragaei** [drscotthawley/panotti#59]
Is it fine if the training and validation loss decrease then increase then decrease?
How to know that my model is not overfitting or underfitting?
